nixos/modules/programs/gphoto2.nix

```diff
@@ -11,12 +14,20 @@ with lib;
       enable = mkOption {
         default = false;
         type = types.bool;
+        description = lib.mdDoc "Whether to configure system to use gphoto2.";
+      };
+      group = lib.mkOption {
         description = lib.mdDoc ''
-          Whether to configure system to use gphoto2.
-          To grant digital camera access to a user, the user must
-          be part of the camera group:
-          `users.users.alice.extraGroups = ["camera"];`
+          Group created when gphoto2 is enabled.
+
+          To grant digital camera access to a user, the user must be part of this group:
+          `users.users.alice.extraGroups = [options.programs.gphoto2.group.value];`.
```

Comment:
Shouldn't this be `config.programs...` instead of `options.programs...`? (I think `options...` is only the option defaults, whereas `config...` reflects the fully merged user config.)

Reply:
I think you're right. Like this?